pandas/core/indexes/accessors.py

```diff
@@ -26,8 +26,8 @@ class Properties(PandasDelegate, PandasObject, NoNewAttributesMixin):
     def __init__(self, data, orig):
         if not isinstance(data, ABCSeries):
             raise TypeError(
-                f"cannot convert an object of type {type(data)} to a "
-                "datetimelike index"
+                "cannot convert an object of type {0} to a "
+                "datetimelike index".format(type(data))
```

Comment:
@EydenVillanueva It looks like the merge undid some changes on master. can you merge again, see https://dev.pandas.io/docs/development/contributing.html#updating-your-pull-request